index.js

```diff
@@ -1615,6 +1629,33 @@
            return either.isRight ? f(x, either.value) : x;
          });
 
+  //# Either#sequence :: Applicative f => Either a (f b) ~> (b -> f b) -> f (Either a b)
+  //.
+  //. Evaluates an applicative action contained within the Either,
+  //. resulting in:
+  //.
+  //.   - a pure applicative of a Left if `this` is a Left; otherwise
+  //.
+  //.   - an applicative of a Right of the evaluated action.
+  //.
```

Comment:
Please confirm that the type signature is correct.

Let me know if you have suggestions as to how to improve the description.